cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  const email = 'mlemeshev@i.ua';
+  const password = 'User1234';
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
```

Comment:
The base URL for the website should ideally be stored in a separate configuration file. This way, if the base URL changes, you only have to update it in one place.